src/lib.rs

```diff
@@ -3688,6 +3706,82 @@ safety_comment! {
     unsafe_impl!(T: ?Sized + Unaligned => Unaligned for UnsafeCell<T>);
     assert_unaligned!(UnsafeCell<()>, UnsafeCell<u8>);
 }
+
+// SAFETY: See safety comment in `is_bit_valid` impl.
+//
+// TODO: Try to add `T: ?Sized` bound.
```

Comment:
TODO: This should be doable, but we'll need to replace `UnsafeCell::get_mut` with a manual implementation.

Reply:
On second thought, we can just do this in a follow-up. It's not a 0.8 blocker.